zcash_client_backend/src/keys.rs

```diff
@@ -28,18 +45,223 @@ pub fn spending_key(seed: &[u8], coin_type: u32, account: u32) -> ExtendedSpendi
         &[
             ChildIndex::Hardened(32),
             ChildIndex::Hardened(coin_type),
-            ChildIndex::Hardened(account),
+            ChildIndex::Hardened(account.0),
         ],
     )
 }
 
+#[cfg(feature = "transparent-inputs")]
+pub fn derive_transparent_address_from_secret_key(
```

Comment:
Document this.

Reply:
I've decided to remove a number of these instead; this proliferation of convenience methods won't be convenient once we fix the UFVK integration as noted by @therealyingtong

Reply:
Fixed in https://github.com/zcash/librustzcash/pull/341/commits/ffc4d0cefbf53b846a5e7ce3dd5f88c45b107861